src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -387,14 +391,29 @@ private IRoleClaimStore<TRole> GetClaimStore()
         ///     Logs the current Identity Result and returns result object
         /// </summary>
         /// <param name="result"></param>
-        /// <param name="user"></param>
+        /// <param name="role"></param>
         /// <param name="methodName"></param>
         /// <returns></returns>
-        protected async Task<IdentityResult> LogResultAsync(IdentityResult result,
+        protected virtual async Task<IdentityResult> LogResultAsync(IdentityResult result,
```

Comment:
Can you refactor this method to also use something like `var logLevel = result.Succeeded ? LogLevel.Information : LogLevel.Warning` etc. as you did in a different version of the method?